### PR TITLE
#12 Feat: Company Overview Section

### PR DESCRIPTION
**Pull request by Lily-Lilian**
### Task Description _(❗️Required)_

Added Company Overview section for all the breakpoints.
The content on the iPad screen and above is aligned to the right. Clicking on each of the three links render a fresh component with the description.
On the small screen the links are displayed in a column clicking on each link displays the description component right below the link.

### Type of Task

- [ ] fix
- [ ] chore
- [x] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [x] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_

https://www.notion.so/apeunit/CompanyOverview-Section-a804d1a5d3ca44fd9bfaff00128c30ea

### CODE REVIEW

File: components/Layout/index.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import Header from './Header'
-
+import CompanyOverview from '../sections/CompanyOverview'
 const Layout = ({children}) => {
   return (
     <div className="font-neuf">
         <Header />
+        <CompanyOverview/>
```
Comment (mercihabam) [inline components/Layout/index.jsx]: This component must not be imported there, instead it has to go in the home component

File: components/sections/CompanyOverview/Overview.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const Overview = ({paragraph}) => {
+    return (
+        <div className="text-xl space-y-7 sm:text-2xl -tracking-tighter sm:h-412">
+            <div className="">{paragraph?.text}</div>
```
Comment (mercihabam) [inline components/sections/CompanyOverview/Overview.jsx]: As the `className` attribute is empty you have to remove it.

File: components/sections/CompanyOverview/index.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    useEffect(()=>{
+        const handleResize = () =>{
+            if(window.innerWidth>= 640&&!activeTabIndex){
+                setActiveTab(data[0])
+                setActiveTabIndex(0)
+            } 
+        } 
+        window.addEventListener("resize", handleResize)
+        return ()=> window.removeEventListener("resize", handleResize)
+    },[activeTab])
```
Comment (mercihabam) [inline components/sections/CompanyOverview/index.jsx]: You have to follow js spacing rules at these statements.

File: components/sections/CompanyOverview/index.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                    {
+                        data.map((item, index) =>
+                            <li className="space-y-4.5" key={index}>
+                                <div onClick={()=>{changeActiveTabMobile(index);setActiveTab(item)}} className={"flex items-center justify-between border-b "+ (activeTabIndex===index?"border-purple":"border-primaryGray-300")}>
+                                    <p className={"cursor-default text-xl -tracking-wider "+(activeTabIndex===index?"text-purple ":"")}> {item?.title==="hiring"?"Employment":item.title}</p>
+                                    <div className={`transition-all duration-300 ease-out ${activeTabIndex===index? "rotate-180": ""}`}>
+                                        <ArrowIcon variant={activeTabIndex===index? " text-purple ": " text-black "}/>
+                                    </div>
+                                </div>
+                                {activeTabIndex===index&&<div className="pb-8.4">
```
Comment (mercihabam) [inline components/sections/CompanyOverview/index.jsx]: Spacing ⚠️

Comment (mercihabam) [inline components/sections/CompanyOverview/index.jsx]: Spacing [⚠️](https://emojipedia.org/warning/)

File: components/sections/CompanyOverview/index.jsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+                    </ul>
+                    <Overview paragraph={activeTab} />
+                </div>
+            </div>          
+            <div className="block sm:hidden">
+                <ul className='space-y-5.1'>
+                    {
+                        data.map((item, index) =>
+                            <li className="space-y-4.5" key={index}>
+                                <div onClick={()=>{changeActiveTabMobile(index);setActiveTab(item)}} className={"flex items-center justify-between border-b "+ (activeTabIndex===index?"border-purple":"border-primaryGray-300")}>
```
Comment (mercihabam) [inline components/sections/CompanyOverview/index.jsx]: Spacing ⚠️

Comment (mercihabam) [inline components/sections/CompanyOverview/index.jsx]: Spacing ⚠️

File: components/sections/CompanyOverview/index.jsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+                <div className="sm:w-full lg:w-757 xl:w-811">
+                    <ul className="flex gap-10.5 items-center py-5 relative">
+                        {
+                            data.map((item, index) =>(
+                                <li className="space-y-5 relative" key={index}>
+                                    <button className={"cursor-pointer text-2xl leading-extra -tracking-tighter transition-all ease-in-out duration-300 "+(index===activeTabIndex?" text-purple ": " text-primaryGray-300 ")} key={index} onClick={() =>{setActiveTabIndex(index); setActiveTab(item)}} id={index}>
+                                      {item.title==="Hiring"?<><span className="hidden lg:block">Hiring</span><span className=" lg:hidden">Employment</span></>
+                                      :<span>{item?.title}</span>}
+                                    </button>
+                                    {index===activeTabIndex?(<Motion.div className="absolute mt-0 bottom-1 inset-x-0 h-0.2 bg-purple" layoutId="underline"/>):null}
```
Comment (mercihabam) [inline components/sections/CompanyOverview/index.jsx]: Spacing ⚠️

**Review by mercihabam — CHANGES_REQUESTED** [review]
## Good job implementing this section

Just some few mistakes I found you have to fix.

File: components/sections/CompanyOverview/index.jsx [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+import ArrowIcon from '../../ui/ArrowIcon';
+import Wrapper from '../Wrapper';
+import Overview from './Overview';
+import LinkItem from './../../ui/LinkItem';
+import {motion as Motion, AnimatePresence } from 'framer-motion';
+const data = [
+    {
+        title: "About",
+        text: <>
+            We believe that blockchain technology has the potential to shift financial models as we know them, creating revenue streams that are more sustainable and fair for users. We envision a future in which financial infrastructures are rather decentralised and horizontal, in which any member can have a say. Through its immutability, blockchain technology provides trustworthy infrastructures that allow cooperation on a large scale.{" "}
```
Comment (clever-cr) [inline components/sections/CompanyOverview/index.jsx]: Remove this empty curly braces to clean codes

**Review by clever-cr — CHANGES_REQUESTED** [review]
Great job!
I left some reviews check it out

**Review by clever-cr — DISMISSED** [review]
LGTM!

File: components/sections/CompanyOverview/index.jsx [hunk 8]
```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import ArrowIcon from '../../ui/ArrowIcon';
+import Wrapper from '../Wrapper';
+import Overview from './Overview';
+import LinkItem from './../../ui/LinkItem';
+import {motion as Motion} from 'framer-motion';
+const data = [
```
Comment (alain-kubwayo) [inline components/sections/CompanyOverview/index.jsx]: The data should be defined in the data folder in the project's root directory.

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good Job!
```
I left a comment

**Review by mercihabam — APPROVED** [review]

**Review by alain-kubwayo — APPROVED** [review]
Looks great! 🚀

**Review by clever-cr — APPROVED** [review]
Great job!

**Pull request merged**